util/src/test/java/tech/pegasys/teku/util/config/KeyStoreFilesLocatorTest.java

```diff
@@ -189,6 +191,27 @@ public void shouldHandleOldArgs(@TempDir final Path tempDir) throws IOException
         .containsExactly(tuple(tempDir, List.of("key", "a"), List.of("pass", "a.txt")));
   }
 
+  @Test
+  public void shouldHandleSymlinkedDirectories(@TempDir final Path tempDir) throws IOException {
+    Path realKeyDir = Path.of("actualKey");
+    Path realPassDir = Path.of("actualPass");
+    createFolders(tempDir, realKeyDir, realPassDir);
+    createFiles(tempDir, realKeyDir.resolve("a.json"), realPassDir.resolve("a.txt"));
+
+    Files.createSymbolicLink(tempDir.resolve("key"), realKeyDir);
+    Files.createSymbolicLink(tempDir.resolve("pass"), realPassDir);
```

Comment:
`createSymbolicLink` isn't supported on all file systems so we probably should catch the `UnsupportedOperationException` it may throw and just skip the test in that case.

```suggestion
    try {
        Files.createSymbolicLink(tempDir.resolve("key"), realKeyDir);
        Files.createSymbolicLink(tempDir.resolve("pass"), realPassDir);
    } catch (UnsupportedOperationException e) {
        throw new AssumptionViolatedException("Symbolic links not supported on this file system");
    }
```

Reply:
Good idea! Looks like that exception class comes from an older junit. junit-jupiter-api has a "TestAbortedException" - let me know if you want me to do something different.